src/views/Target.vue

```diff
@@ -225,7 +210,7 @@ async function get_db_info() {
         return
     }
 
-    await axios.get(import.meta.env.VITE_API_URL + '/info/database')
+    await apiInstance.get(import.meta.env.VITE_API_URL + '/info/database')
```

Comment:
I guess we can now remove the base url here?

Reply:
Done